ws-nextjs-app/pages/[service]/live/[id]/Header/index.tsx

```diff
@@ -36,24 +28,17 @@ const Header = ({
           {/*  eslint-disable-next-line jsx-a11y/aria-role */}
           <span role="text" css={styles.innerGrid}>
             {showLiveLabel ? (
-              <>
-                <span
-                  css={styles.label}
-                  aria-hidden="true"
-                  data-testid="live-label"
-                >
-                  {liveLabel}
-                </span>
-                <VisuallyHiddenText lang="en-GB">
-                  {`${liveLabel}, `}
-                </VisuallyHiddenText>
-              </>
-            ) : null}
-            <span
-              css={[styles.title, showLiveLabel && styles.layoutWithLiveLabel]}
-            >
-              {title}
-            </span>
+              <LiveLabel>{title}</LiveLabel>
```

Comment:
What would it look like if we didn't put the title as a child of the component, but rather render it on its own.

```suggestion
              <LiveLabel />
```

Reply:
What if we also used the (clatty hack) first-promo class:

```suggestion
              <LiveLabel className='first-promo' >
```

Reply:
We need it nested as a child for screen reader/a11y purposes I think https://github.com/bbc/simorgh/blob/92dc6472330112d78af808bd77b30adca4aa371f/src/app/components/LiveLabel/index.tsx#L71-L77

Again it feels weird adding first-promo when this isn't a promo, we could rename the class, but again it adds complexity to the live label component for specific live page styling

Reply:
Yeah true, we could rename the class to be better.. but like you said, it's starting to add more and more complexity to the component. Just for completeness, can we add the first-promo class to see if it matches the live page designs? Or are we just wasting time going down this road trying to make it work?

Reply:
So the [design](https://www.figma.com/file/ozHsWG5R9tETw6lBOU740V/Live-mvp-header-(text-only)---handover?node-id=402%3A37066&mode=dev) has a width and height of 1.5rem, which is 24px for all breakpoints which is slightly different from first-promo

Reply:
Ah OK then so, thanks for explaining. Let's just bin this, and focus on the other PR. Thanks for trying this out, and apologies for making you do extra work. I just wanted to make sure that we had confirmed 100% that using the label as is was not going to be suitable before committing to the refactor.